service/dap/server.go

```diff
@@ -625,22 +622,25 @@ func (s *Server) onAttachRequest(request *dap.AttachRequest) { // TODO V0
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
-// onNextRequest sends a not-yet-implemented error response.
+// onNextRequest handles 'next' request.
 // This is a mandatory request to support.
-func (s *Server) onNextRequest(request *dap.NextRequest) { // TODO V0
-	s.sendNotYetImplementedErrorResponse(request.Request)
+func (s *Server) onNextRequest(request *dap.NextRequest) {
+	s.send(&dap.NextResponse{Response: *newResponse(request.Request)})
+	s.doCommand(api.Next, "step")
```

Comment:
It's pretty confusing that "step" is propagated all the way from here just to have `doCommand` and donwstream methods more general. It's not clear what "step" means here.

I think this could be clearer if there was a switch statement in `handleStop` that would take the command and infer the reason string from it. This would keep the reason closer to where it's needed and remove the need to push it through several layers. WDYT?

Reply:
The code has been refactored and simplified. The mapping between dap command and api command is 1-1, so mapping to reasons is reliable.